### PR TITLE
Isenthalpic compressibility added to properties.

### PR DESCRIPTION
**Pull request by RolfStierle**
Add `isenthalpic_compressibility()`, `themal_expansivity()`, and `grueneisen_parameter()` to the properties that can be calculated for each `State``.

### CODE REVIEW

**Comment by prehner** [comment]
Thanks for the nice addition! Can you also amend the section in the theory guide?

**Comment by RolfStierle** [comment]
I added the three properties to the theory guide under Theory/Equation of state/Properties and amended this pull request.

File: feos-core/src/python/state.rs [hunk 1]
```diff
@@ -903,6 +903,33 @@ macro_rules! impl_state {
...
+            /// SINumber
+            fn thermal_expansivity(&self) -> PySINumber {
+                PySINumber::from(self.0.thermal_expansivity())
+            }
+
+            /// Return Grueneisen parameter.
+            ///
+            /// Returns
+            /// -------
+            /// SINumber
```
Comment (prehner) [inline feos-core/src/python/state.rs:928]:
```suggestion
            /// float
```

**Review by prehner — APPROVED** [review]

**Comment by prehner** [comment]
Ready to merge?

**Comment by RolfStierle** [comment]
Ready

**Pull request merged**